src/Init/Notation.lean

```diff
@@ -551,6 +551,8 @@ macro_rules
 /--
 Haskell-like pipe operator `|>`. `x |> f` means the same as `f x`,
 and it chains such that `x |> f |> g` is interpreted as `g (f x)`.
```

Comment:
This is a strict improvement as-is.

However, while you're at it, perhaps something like this this would be good too?
```suggestion
The pipe operator `|>`, which allows a function to be placed _after_ its argument.

`x |> f` is equivalent to `f x`, and it chains such that `x |> f |> g` is interpreted as `g (f x)`. This
directs readers' attention to an initial value and a sequence of transformations, instead of
centering the function.
```
This operator is not widely used in Haskell, where `&` is more common for this purpose. It's certainly very common in F#, and I believe it's also popular in many Haskell-derived languages. I think it's also useful to point out why someone might want to write the argument before the function.